google/cloud/forseti/services/cli.py

```diff
@@ -113,16 +129,17 @@ def define_inventory_parser(parent):
     create_inventory_parser = action_subparser.add_parser(
         'create',
         help='Start a new inventory')
+    create_inventory_parser.add_argument(
+        'import_as',
```

Comment:
Why are we dropping the dash dash usage here `'--import_as'```?

Reply:
This is now becoming a positional parameter and not a flag any more, -- doesn't seem needed